components/DashboardPage/components/HabitTracker/Habit/Habit.tsx

```diff
@@ -0,0 +1,45 @@
+import { Stack, Typography } from 'components';
+import { ChangeEventHandler, useState } from 'react';
+import { Checkbox } from '../Checkbox';
+import styles from './Habit.module.scss';
+
+interface HabitProps {
+  label: string;
+  days: boolean[];
+  onChange: any;
+  onChecked: any;
+}
+
+export function Habit({
+  label,
+  days,
+  onChange,
+  onChecked,
+}: HabitProps) {
+  return (
+    <Stack>
+      <Stack width="25%">
+        <input
+          type="text"
+          placeholder="Add a habit"
+          className={styles.Text}
+          onChange={onChange}
+          value={label}
+        />
+      </Stack>
+      <Stack width="75%">
+        <Stack width="100%" spreadEvenly>
+          {days.map(
+            (isChecked: boolean, dayIndex: number) => (
+              <Checkbox
+                key={dayIndex}
```

Comment:
not a good idea to use index directly for a key. `id=1` likely won't be unique on the webpage.